projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1615,17 +1615,10 @@ public function is_enhanced_publishing_enabled( $blog_id ) {
 	/**
 	 * Check if the social image generator is enabled.
 	 *
-	 * @param int $blog_id The blog ID for the current blog.
 	 * @return bool
 	 */
-	public function is_social_image_generator_enabled( $blog_id ) {
-		$data = $this->get_api_data( $blog_id );
-
-		if ( empty( $data ) ) {
-			return false;
-		}
-
-		return ! empty( $data['is_social_image_generator_enabled'] );
+	public function has_social_image_generator_feature() {
```

Comment:
We've renamed this function as it gets overridden on WPCOM, and the signature changing was causing problems.